distribution/docs/sender/application.properties

```diff
@@ -37,6 +37,24 @@ db-event.retry.initial.delay=120000
 
 # Number of events to sync in parallel, defaults to 10
 events.sync.parallel.size=
+
+# The interval in milliseconds before the first sender sync messages fetch by the active mq publisher route when the application starts.
+active-mq-publisher.initial.delay=120000
+
+# The interval in milliseconds between sender sync messages fetches by the active mq publisher route.
+active-mq-publisher.delay=60000
+
+# Maximum results of each fetch of sender sync messages to be read by active mq publisher route.
+active-mq-publisher.max.fetch.results=500
+
+# The interval in milliseconds before the first sender sync responses fetch by the sync response processor route when the application starts.
+sync-response-processor.initial.delay=120000
+
+# The interval in milliseconds between sender sync responses fetches by the sync response processor route.
+sync-response-processor.delay=60000
+
+# Maximum results of each fetch of sender sync responses to be read by sync response processor route.
+sync-response-processor.max.fetch.results=500
 # ----------------------------------------------------------------------------------------------------------------------
 
```

Comment:
Please add the loggers for the new routes too and if you renamed any new routes update their loggers to match the new route names